diff --git a/node_modules/opensea-js/lib/index.d.ts b/node_modules/opensea-js/lib/index.d.ts
index 5b313d4..5551f45 100644
--- a/node_modules/opensea-js/lib/index.d.ts
+++ b/node_modules/opensea-js/lib/index.d.ts
@@ -1,7 +1,7 @@
 import { OpenSeaPort } from './seaport';
 import { OpenSeaAPI } from './api';
 import { Network, EventData, EventType } from './types';
-export { orderToJSON, orderFromJSON, WyvernProtocol } from './utils/utils';
+export { orderToJSON, orderFromJSON, WyvernProtocol, assetFromJSON } from './utils/utils';
 export { encodeCall, encodeSell, encodeAtomicizedBuy, encodeAtomicizedSell, encodeDefaultCall, encodeReplacementPattern, AbiType, } from './utils/schema';
 /**
  * Example setup:
diff --git a/node_modules/opensea-js/lib/index.js b/node_modules/opensea-js/lib/index.js
index 35b2467..2dce423 100644
--- a/node_modules/opensea-js/lib/index.js
+++ b/node_modules/opensea-js/lib/index.js
@@ -11,6 +11,7 @@ Object.defineProperty(exports, "EventType", { enumerable: true, get: function ()
 var utils_1 = require("./utils/utils");
 Object.defineProperty(exports, "orderToJSON", { enumerable: true, get: function () { return utils_1.orderToJSON; } });
 Object.defineProperty(exports, "orderFromJSON", { enumerable: true, get: function () { return utils_1.orderFromJSON; } });
+Object.defineProperty(exports, "assetFromJSON", { enumerable: true, get: function () { return utils_1.assetFromJSON; } });
 Object.defineProperty(exports, "WyvernProtocol", { enumerable: true, get: function () { return utils_1.WyvernProtocol; } });
 var schema_1 = require("./utils/schema");
 Object.defineProperty(exports, "encodeCall", { enumerable: true, get: function () { return schema_1.encodeCall; } });
